samples/net/rpl-node/README.rst

```diff
@@ -0,0 +1,72 @@
+.. _rpl-node-sample:
+
+RPL node
+###########
+
+Overview
+********
+
+A simple RPL node showing how to join in RPL mesh network.
+
+This demo assumes that the platform of choice has networking support,
```

Comment:
Change to:

     This sample assumes that your chosen platform has networking support.
     Some code configuration adjustments may be needed.